pkg/storage/stores/tsdb/index.go

```diff
@@ -22,6 +24,18 @@ type ChunkRef struct {
 	Checksum    uint32
 }
 
+type IndexCfg struct {
```

Comment:
Can we just name it `Config`?

Reply:
I think we can, but wouldn't it mislead you into thinking it is a general TSDB configuration? While this one is only relevant in the context of the index.